ibis-server/app/routers/ibis/bigquery.py

```diff
@@ -21,3 +23,17 @@ def query(dto: BigQueryDTO, dry_run: Annotated[bool, Query(alias="dryRun")] = Fa
         connector.dry_run(dto.sql)
         return Response(status_code=204)
     return JSONResponse(connector.query(dto.sql))
+
+
+@router.post("/metadata/tables")
+@log_dto
+def get_bigquery_table_list(dto: MetadataDTO) -> dict:
+    table_list = Metadata.bigquery.get_table_list(dto.connection_info)
+    return {"tables": table_list}
+
+
+@router.post("/metadata/constraints")
+@log_dto
+def get_bigquery_constraints(dto: MetadataDTO) -> dict:
+    table_list = Metadata.bigquery.get_constraints(dto.connection_info)
+    return {"constraints": table_list}
```

Comment:
It doesn't have any other data. How about just returning a list to a JSON array?
BTW, plz add a empty line in the end of file.